src/db/config/config.js

```diff
@@ -0,0 +1,39 @@
+require('dotenv').config()
+const fs = require('fs');
+
+module.exports = {
+  development: {
+    username: process.env.DB_USER,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_NAME,
+    host: '127.0.0.1',
+    port: 5432,
+    dialect: 'postgres',
+    dialectOptions: {
+      bigNumberStrings: true
+    }
+  },
+  test: {
+    username: process.env.CI_DB_USERNAME,
+    password: process.env.CI_DB_PASSWORD,
+    database: process.env.CI_DB_NAME,
+    dialect: 'postgres',
+    dialectOptions: {
+      bigNumberStrings: true
+    }
+  },
+  // production: {
+  //   username: process.env.PROD_DB_USERNAME,
+  //   password: process.env.PROD_DB_PASSWORD,
+  //   database: process.env.PROD_DB_NAME,
+  //   host: process.env.PROD_DB_HOSTNAME,
+  //   port: process.env.PROD_DB_PORT,
+  //   dialect: 'postgres',
+  //   dialectOptions: {
+  //     bigNumberStrings: true,
+  //     ssl: {
+  //       ca: fs.readFileSync(__dirname + '/mysql-ca-main.crt')
+  //     }
+  //   }
+  // }
```

Comment:
You can remove these commented source codes, anyways the work looks great 👌

Reply:
Hi @aimelyse, in your package.json there are some typescript dependencies which are causing some errors. So, I think if we are completely not using typescript, you may remove it.